pkg/cloud-controller-manager/ccm.go

```diff
@@ -30,7 +36,7 @@ func newCloudProvider(reader io.Reader) (cloudprovider.Interface, error) {
 	if err != nil {
 		return nil, err
 	}
-	clientConfig, err := config.ClientConfig()
+	ClientConfig, err = config.ClientConfig()
```

Comment:
Generally, the variable name starts with a lower letter.

Reply:
The variable is global and the controller uses the variable. pkg/controller/setup.go#23